ROADMAP.md

```diff
@@ -5,6 +5,7 @@
 Objective: "Simplify the user experience and provide a low barrier to entry by minimizing the amount of YAML necessary to deploy a trained model."
 * High Level Interfaces
     * Deploy a Tensorflow model without specifying a Tensorflow Serving Technology.
+    * Deploy a TensorRT model without specifying a TensorRT Serving Technology.
```

Comment:
Is there a model of 'type' TensorRT? We have TF, PyTorch, scikit-learn etc....seems a bit odd to say TensorRT 'model'.

Reply:
TensorRT has a model format called "plan" - but it is GPU and cuda version specific, which makes me think it's weird for this level of abstraction.

See https://docs.nvidia.com/deeplearning/sdk/tensorrt-developer-guide/index.html#work

Reply:
updated the text to not call it a model